lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1662,6 +1670,45 @@ def cmd_db_nmap(*args)
   }
   end
 
+  def cmd_db_nmap_help
+    nmap =
+        Rex::FileUtils.find_full_path('nmap') ||
+        Rex::FileUtils.find_full_path('nmap.exe')
+
+    stdout, stderr = Open3.capture3([nmap, 'nmap'], '--help')
+
+    stdout.each_line do |out_line|
+      next if out_line.strip.empty?
+      print_status(out_line.strip);
```

Comment:
Yes I have, as a matter of fact.  Related: I probably won't get around to the requested changes until this weekend.  Sorry for the delay!